app/models/lesson.rb

```diff
@@ -0,0 +1,22 @@
+class Lesson < ApplicationRecord
+  belongs_to :course
+
+  has_one_attached :video
+  has_one_attached :content
+
+  validates :title, presence: true
+  validates :order, presence: true
+  validates :video, content_type: { in: ['video/mp4', 'video/webm', 'video/ogg'], message: 'must be a video file (MP4, WebM, or Ogg)' }
+
+  def video_duration
```

Comment:
why you need this?

Reply:
Apu, actually Sarah apu suggested this to add validation: https://github.com/amu-wtag/well_academy/pull/2#pullrequestreview-2432547687 for the type of videos users can upload.